index.html

```diff
@@ -82,6 +82,73 @@ <h1>I’m Azaria<br>
                 </ul>
             </div>
         </section>
```

Comment:
- [ ] It would be better if we include the `works` section. This section contains all the projects, with each project aligned with flexbox, has a title, has an image, a paragraph `<p>`, button with the styling, project technologies, etc

Reply:
I don't understand when you speak about the works section, should I put a class named Works in the section tag?

Reply:
if you were speaking about a general container  I've it already